docs/src/pages/components/drawers/MiniDrawer.js

```diff
@@ -49,6 +49,41 @@ const DrawerHeader = styled('div')(({ theme }) => ({
   ...theme.mixins.toolbar,
 }));
 
+const AppBar = styled(MuiAppBar, {
+  shouldForwardProp: (prop) => prop !== 'open',
```

Comment:
How cool this shape! I've been reading the documentation for `shouldForwardProp`.

Some time ago I ran into this problem and ended up using destructuring to not pass the prop to the DOM.